proxy/invoke.go

```diff
@@ -133,3 +138,23 @@ func parseHeaders(headers []string) (map[string]string, error) {
 	}
 	return headerMap, nil
 }
+
+// validateMethod validates the HTTP request method
+func validateMethod(method string) (string, error) {
```

Comment:
maybe we should refer to `method` as `httpMethod` everywhere to avoid ambiguity and keep it as `--method` just for the flag (user-facing)?